### ISSUE TITLE
Unerklärlicher Vorfruchtwert in der Bilanz enthalten

### ISSUE THREAD

**Issue by simonkriegnerschramml** [w3geo/bml-fast#141]
Hallo Arnold,

auf dem Feld "Test2 - Hauptkultur und eine genutzte ZWF" gibt es bei der Bilanz der Sommerzwischenfrucht den Eintrag "Vorfruchtwert Vorfrucht" von 10 kg. Es ist aber gar keine Vorfrucht eingetragen. Warum kommt es zu diesem Wert? Woher stammt dieser?

LG
Simon

[fast-export-2024-12-11.txt](https://github.com/user-attachments/files/18096313/fast-export-2024-12-11.txt)

**Comment by arnold-pichler** [comment]
Sehe schon, der VFW der Zwischenfrucht wird da doppelt verbucht, einmal als VFW und einmal VFW der Zwischenfrucht.

**Comment by arnold-pichler** [comment]
> Zur Info: bei ungenutzten Zwischenfrüchten >60% Leguminosen passiert das nicht. Aber probiere selbst aus.

Da werden aber auch die 20 als Vorfruchtwert Vorfrucht angerechnet. Ist das korrekt? Müsste das dann nicht wenn, der Vorfruchtwert der Zwischenfrucht sein?

![image](https://github.com/user-attachments/assets/75f35718-7990-482e-9234-4d51a4159a3f)

**Comment by arnold-pichler** [comment]
OK, das habe ich gefunden und war im Grunde nur ein Tippfehler. Jetzt ist es so: 

Genutzte ZF:
- Der Wert der Vorfrucht geht als Vorfruchtwert der VF in die Bilanz der genutzten Zwischenfrucht
- Der Wert der Zwischenfrucht geht als Vorfruchtwert der Zwischenfrucht in die Bilanz der Hauptfrucht

Ungenutzte ZF:
- Der Wert der Vorfrucht geht als Vorfruchtwert der VF in die Bilanz der Hauptfrucht
- Der Wert der Zwischenfrucht geht als Vorfruchtwert der Zwischenfrucht in die Bilanz der Hauptfrucht


Korrekt?

**Comment by simonkriegnerschramml** [comment]
> > Zur Info: bei ungenutzten Zwischenfrüchten >60% Leguminosen passiert das nicht. Aber probiere selbst aus.
> 
> Da werden aber auch die 20 als Vorfruchtwert Vorfrucht angerechnet. Ist das korrekt? Müsste das dann nicht wenn, der Vorfruchtwert der Zwischenfrucht sein?
> 
> ![image](https://private-user-images.githubusercontent.com/36012148/395494132-75f35718-7990-482e-9234-4d51a4159a3f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQwODM4NTcsIm5iZiI6MTczNDA4MzU1NywicGF0aCI6Ii8zNjAxMjE0OC8zOTU0OTQxMzItNzVmMzU3MTgtNzk5MC00ODJlLTkyMzQtNGQ1MWE0MTU5YTNmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjEzVDA5NTIzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdiODIwYmY2ODM2NDlkNTUxMjU3OGE1MTM1ZDBjMDc5ZTgyOTUyODBjNTViNWFmNTg5NWNmODQ1ZDhjYjEyMDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.K7ZFKPbk--IVrQ7edw26z3VOcD30qX3XqIXZn1Qe7JY)

Stimmt, das habe ich übersehen. Das müsst dann der VFW der Zwischenfrucht sein.
Die 20 kg stimmen, aber die Zuordnung zur verursachenden Kultur eben nicht.
Danke für den aufmerksamen Blick

LG
Simon

**Comment by simonkriegnerschramml** [comment]
> OK, das habe ich gefunden und war im Grunde nur ein Tippfehler. Jetzt ist es so:
> 
> Genutzte ZF:
> 
> * Der Wert der Vorfrucht geht als Vorfruchtwert der VF in die Bilanz der genutzten Zwischenfrucht
> * Der Wert der Zwischenfrucht geht als Vorfruchtwert der Zwischenfrucht in die Bilanz der Hauptfrucht
> 
> Ungenutzte ZF:
> 
> * Der Wert der Vorfrucht geht als Vorfruchtwert der VF in die Bilanz der Hauptfrucht
> * Der Wert der Zwischenfrucht geht als Vorfruchtwert der Zwischenfrucht in die Bilanz der Hauptfrucht
> 
> Korrekt?

Exakt so soll es sein.

LG
Simon

**Comment by arnold-pichler** [comment]
Sollte jetzt passen!

**Comment by simonkriegnerschramml** [comment]
Gilt als gefixed. Ich schließe das Issue.

LG
Simon
